plugins/kn-curl/kn-curl

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
maybe we could restrict to a simple bourne shell to make it the plugin more conservative and allow it to run on more platforms (e.g. also on Alpine which only has `ash` by default) ?

This would require to avoid bash specific extensions, but since this plugin is quite simple I think this should be no issue.

Reply:
Did you really mean `ash`?

Reply:
@maximilien : `/bin/sh`